### PR TITLE
Rename trackAjax to trackDependency

### PR DESCRIPTION
**Pull request by kamilszostak**
#196

### CODE REVIEW

**Comment by MaxShehovtsov** [comment]
Did you change the snippet? I also wonder if we can do this change as snippet has trackAjax and potentially someone could already start using it explicitely. If we change the snippet and/or the implementation they'll be broken.

File: API-reference.md [hunk 1]
```diff
@@ -155,6 +155,23 @@ In the portal, you can search on message content and [display individual trackTr
 (Unlike `trackEvent`, you can't filter on the message content in the portal.)
 
 
+### trackDependency
+
+    trackDependency(id: string, method: string, absoluteUrl: string, pathName: string, totalTime: number, success: boolean, resultCode: number) {
+    
+Log an ajax dependency
```
Comment (Dmitry-Matveev) [inline API-reference.md]: Should we consider changing this from "Log an ajax dependency" to just "dependency" then.

**Comment by dnduffy** [comment]
:shipit:

**Comment by kamilszostak** [comment]
@MaxShehovtsov Yes, I updated the snippet. I left `trackAjax` method so existing costumers won't be impacted. But the method is not exposed in `IAppInsights` interfaces or *.d.ts files. This should force users to switch to `trackDependency` after they update the nuget package and try to build their solution. 

Once we release this change we will have to update the snippet code in the JS SDK on-boarding page.

**Comment by MaxShehovtsov** [comment]
:shipit:

**Pull request merged**